Formula/tnftp.rb

```diff
@@ -0,0 +1,38 @@
+class Tnftp < Formula
+  desc "NetBSD's FTP client (built from macOS Sierra sources)"
+  homepage "https://opensource.apple.com/"
+  url "https://opensource.apple.com/tarballs/lukemftp/lukemftp-16.tar.gz"
+  sha256 "ba35a8e3c2e524e5772e729f592ac0978f9027da2433753736e1eb1f1351ae9d"
+
+  depends_on :xcode => :build
+
+  conflicts_with "inetutils", because: "both install `ftp' binaries"
+
+  def install
+    # Trying to use Apple's pre-supplied Makefile resulted
+    # in headaches... they have made the build process
+    # specifically for installing to /usr/bin and so it
+    # just doesn't play well with homebrew.
+
+    # so just build straight from ftp's own sources
+    Dir.chdir "tnftp"
```

Comment:
`cd "tnftp" do`

Reply:
@DomT4 Is there some reason that is preferable to `Dir.chdir`?

Reply:
Just more the preferred usage unless there's a particularly good reason for doing otherwise.

Reply:
@DomT4 Ok, well I would change it but I'm having some really weird git problems on my end so I can't really push or pull to my fork for the time being. If you could fix it to do that, I'd appreciate it.

Reply:
This can be tweaked now if you have git access/function back.